.github/workflows/build.yml

```diff
@@ -44,6 +44,10 @@ jobs:
         uses: Jerome1337/goimports-action@v1.0.3
         with:
           goimports-path: ./litmus-portal
+      - uses: Jerome1337/gofmt-action@v1.0.4
+        with:
+          gofmt-path: './litmus-portal'
+          gofmt-flags: '-l -d'
```

Comment:
gofmt check is already present

Reply:
In the Makefile

Reply:
Sir but there was some unformatted code , which got formatted when i ran the gofmt command from the terminal

Reply:
I am not sure is , it because of the go fmt version ?

Reply:
Should i remove it then ?